includes/frontend.php

```diff
@@ -541,10 +541,10 @@ public function register_styles() {
 		);
 
 		wp_register_style(
-			'swiper',
-			$this->get_css_assets_url( 'swiper', $this->e_swiper_asset_path . 'css/' ),
-			[],
-			$this->e_swiper_version
+			'e-swiper',
+			$this->get_css_assets_url( 'e-swiper', 'assets/css/conditionals/' ),
+			[ 'swiper' ],
+			ELEMENTOR_VERSION
```

Comment:
Question: why did you deleted the original `swiper` ?

Reply:
Restored.